service/frontend/operator_handler.go

```diff
@@ -696,6 +474,249 @@ func (h *OperatorHandlerImpl) ListClusters(
 	}, nil
 }
 
+func (h *OperatorHandlerImpl) ResendWorkflowHistory(
+	ctx context.Context,
+	request *operatorservice.ResendWorkflowHistoryRequest,
+) (_ *operatorservice.ResendWorkflowHistoryResponse, retError error) {
+	defer log.CapturePanic(h.logger, &retError)
+	scope, startTime := h.startRequestProfile(metrics.OperatorResendWorkflowHistory)
+	defer func() { scope.Timer(metrics.ServiceLatency.GetMetricName()).Record(time.Since(startTime)) }()
+
+	if request == nil {
+		return nil, errRequestNotSet
+	}
+
+	if _, err := h.historyClient.GenerateLastHistoryReplicationTasks(ctx, &historyservice.GenerateLastHistoryReplicationTasksRequest{
```

Comment:
We only resend history on the current branch? 
Looks like the original one is kinda more powerful and can be used to address more replication issues.

So this new api, is it more for oncall issue mitigation or normal operations?

Reply:
ok. I think you make a good point here. My plan was to simplify this operation. But it seems like this will limit to resend on current branch.